api/tests/unit/domain/usecases/authentication/authenticate-oidc-user_test.js

```diff
@@ -34,11 +41,65 @@ describe('Unit | UseCase | authenticate-oidc-user', function () {
       };
 
       userRepository = { findByExternalIdentifier: sinon.stub() };
+      adminMemberRepository = {
+        get: sinon.stub(),
+      };
       userLoginRepository = {
         updateLastLoggedAt: sinon.stub().resolves(),
       };
     });
 
+    context('check access by audience', function () {
+      context('when audience is pix-admin', function () {
+        it('should throw an error when user has no role and is therefore not an admin member', async function () {
```

Comment:
```suggestion
        it('throws an error', async function () {
```

Il faudrait ajouter un contexte en plus avec `when user has no role and is therefore not an admin member`